core/src/main/java/org/apache/iceberg/ContentFileParser.java

```diff
@@ -164,6 +165,7 @@ static ContentFile<?> fromJson(JsonNode jsonNode, PartitionSpec spec) {
     List<Long> splitOffsets = JsonUtil.getLongListOrNull(SPLIT_OFFSETS, jsonNode);
     int[] equalityFieldIds = JsonUtil.getIntArrayOrNull(EQUALITY_IDS, jsonNode);
     Integer sortOrderId = JsonUtil.getIntOrNull(SORT_ORDER_ID, jsonNode);
+    Long firstRowId = JsonUtil.getLongOrNull(FIRST_ROW_ID, jsonNode);
```

Comment:
Are we missing the corresponding `toJson` here?

Reply:
Fixed.